examples/animation/animate_decay.py

```diff
@@ -14,9 +14,7 @@
 
 
 def data_gen(t=0):
-    cnt = 0
-    while cnt < 1000:
-        cnt += 1
+    for cnt in range(1000):
         t += 0.1
```

Comment:
```suggestion
        t = cnt / 10
```
(because of floating point addition errors)

Reply:
fixed, thanks